<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>666</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>666</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AI-Assistant v0.3.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: #2E2E2E;
    }
    QTabWidget::pane {
        border: 1px solid #555;
        background: #3A3A3A;
    }
    QTabBar::tab {
        background: #555;
        color: white;
        padding: 8px;
    }
    QTabBar::tab:selected {
        background: #777;
    }
    QTextEdit {
        background-color: #1E1E1E;
        color: white;
        border: 1px solid #555;
        padding: 5px;
    }
    QPushButton {
        background-color: #4CAF50;
        color: white;
        border: none;
        padding: 8px;
        border-radius: 4px;
    }
    QPushButton:hover {
        background-color: #45a049;
    }
    QLineEdit, QComboBox {
        background-color: #333;
        color: white;
        border: 1px solid #555;
        padding: 5px;
        border-radius: 4px;
    }
    QLabel {
        color: white;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Main Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="mainPageLayout">
        <item>
         <widget class="QTextEdit" name="statusTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;Ready&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputLineEdit">
          <property name="placeholderText">
           <string>Type here...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="consoleTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="recordButton">
            <property name="text">
             <string>Start Recording</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendTextButton">
            <property name="text">
             <string>Send Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="listMicrophonesButton">
            <property name="text">
             <string>List Microphones</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="showLogButton">
            <property name="text">
             <string>Show Log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scanNetworkButton">
            <property name="text">
             <string>Scan Network</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="settingsLayout">
        <item>
         <layout class="QHBoxLayout" name="microphoneLayout">
          <item>
           <widget class="QLabel" name="microphoneLabel">
            <property name="text">
             <string>Microphone:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="microphoneComboBox">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="logPathLayout">
          <item>
           <widget class="QLabel" name="logPathLabel">
            <property name="text">
             <string>Log Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="logPathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="wavPathLayout">
          <item>
           <widget class="QLabel" name="wavPathLabel">
            <property name="text">
             <string>WAV Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="wavPathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="outputTxtPathLayout">
          <item>
           <widget class="QLabel" name="outputTxtPathLabel">
            <property name="text">
             <string>Output TXT Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputTxtPathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="silenceDbLayout">
          <item>
           <widget class="QLabel" name="silenceDbLabel">
            <property name="text">
             <string>Silence dB:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="silenceDbLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="whisperCliPathLayout">
          <item>
           <widget class="QLabel" name="whisperCliPathLabel">
            <property name="text">
             <string>Whisper CLI Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="whisperCliPathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="whisperModelPathLayout">
          <item>
           <widget class="QLabel" name="whisperModelPathLabel">
            <property name="text">
             <string>Whisper Model Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="whisperModelPathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="koboldcppLinkLayout">
          <item>
           <widget class="QLabel" name="koboldcppLinkLabel">
            <property name="text">
             <string>Koboldcpp Link:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="koboldcppLinkLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="koboldcppPathLayout">
          <item>
           <widget class="QLabel" name="koboldcppPathLabel">
            <property name="text">
             <string>Koboldcpp Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="koboldcppPathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="koboldcppCfgPathLayout">
          <item>
           <widget class="QLabel" name="koboldcppCfgPathLabel">
            <property name="text">
             <string>Koboldcpp Config Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="koboldcppCfgPathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="koboldcppModelPathLayout">
          <item>
           <widget class="QLabel" name="koboldcppModelPathLabel">
            <property name="text">
             <string>Koboldcpp Model Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="koboldcppModelPathLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dcSubnetLayout">
          <item>
           <widget class="QLabel" name="dcSubnetLabel">
            <property name="text">
             <string>DC Subnet:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dcSubnetLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dcPortLayout">
          <item>
           <widget class="QLabel" name="dcPortLabel">
            <property name="text">
             <string>DC Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dcPortLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="saveSettingsButton">
          <property name="text">
           <string>Save Settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>716</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
   </widget>
   <resources/>
   <connections/>
  </ui>
